JC-2194 Got rid of drop-down menu of "Code" button

Sabfir opened the pull request:
There is no more "Add syntax higlight" button described in the topic of
this task in Jira, this button's name is "Code".
Got rid of drop-down menu because it does not matter what programming
language we choose in the drop-down list of button "Code". The syntax will
be displayed correct depending on what language it is written.
Pressing button "Code" you will have this construction: [code=]Insert your
text here[/code]
Still old version of constraction e.g [code=java]Insert your text
here[/code] is working correctly too.
modified files:
  jcommune-view/jcommune-web-view/src/main/webapp/WEB-INF/tags/bbeditor.tag
  jcommune-view/jcommune-web-view/src/main/webapp/resources/javascript/lib/wysiwyg-bbcode/editor.js

ctapobep commented:
According to your commit message you're going to print `[code=]Insert your
text here[/code]`, why do we need `=` in the `[code=]`? Can we get rid of it?

Sabfir commented:
Yes, we can!
I changed it to [code]...[/code]

ctapobep commented:
Cool, thanks! Though now you an extra-commit about tooltips in this PR :)

Sabfir commented:
Sorry, I'm not sure I understood your last message.
Have I done everything correctly?

ctapobep commented:
You've got extra commit in this PR:
> JC-2245 Added tooltip(en, ru, uk) on mouse over New poll Ending date

This PR should include only JC-2194-related code.

Sabfir commented:
Ok, I'll try to solve it as soon as possible

Sabfir commented:
I did it)

ctapobep commented:
When you're in the editor in a Discussion and press Code button - it doesn't work anymore. Please make sure that the functionality that was working previously still works.

Sabfir commented:
I can't find this bug.
Could you send me screenshot with the steps to replicate this error?
![image](https://cloud.githubusercontent.com/assets/12494460/13372654/989be6a2-dd55-11e5-8152-b6231c12c9fd.png)

ctapobep commented:
Pretty same steps as yours. Browser: Chrome. Are you building the artifact with Maven or running the app from IntelliJ? There might be differences because of the wro4j JS processing. I'm running it from IntelliJ.

Sabfir commented:
I am running the project from my Intellij.
Browser: Chrome.
I have tested it in firefox - it works correctly as well
Could you connect to my pc via Skype or TeamViewer to test it?

Sabfir commented:
Corrected file "bbeditor.vm" as we agreed before.
Found and fixed a related test failrue. File: BBCodeServiceTest.java
Everything works corrently from the Intellij.
Also unrelated to my task bug was found when I tried to "mvn package" the project.
Here is a screenshot and an attachment with log from command line of the test failure:
![image](https://cloud.githubusercontent.com/assets/12494460/13376795/d7aadaa2-ddcc-11e5-8af4-51a94ffc904e.png)
[test failure 2.txt](https://github.com/jtalks-org/jcommune/files/149692/test.failure.2.txt)

ctapobep commented:
Could you please squash the commits into 1?

As per a test failure - recently we committed the fix, so it should pass if you merge upstream/master into your local branch.

Sabfir commented:
I squashed all my commits from the branch JS-2194 to the one commit
